### ISSUE TITLE
Wrap text/* Media Types in <pre> tags.

### ISSUE THREAD

**Issue by BigBlueHat** [hypothesis/via#3]
Otherwise you can't annotate `text/plain` or `text/csv` like you can in the Firefox and Chrome extensions and the bookmarklet.

**Comment by tilgovi** [comment]
We can?? o_O

**Comment by csillag** [comment]
> We can?? o_O

Sure we can.

Check http://www.textfiles.com/humor/101nos.txt with either FF (with bookmarklet) or Chrome extension.

**Comment by tilgovi** [comment]
Oh. I see. FF makes a pre-wrapped text document out of it and then we inject into that HTML document.

**Comment by csillag** [comment]
Same with Chrome; the only difference is the FF seems to use a SPAN, while Chrome a PRE.

**Comment by ikreymer** [comment]
Yes, I think browsers implicitly convert text content into an html document.
However, pywb injection currently only runs on text/html, so need to add a way to override this and serve html with text wrapped in <pre> and a <head> tag. So basically text as if it were html I think

**Comment by ikreymer** [comment]
Not to get too carried away with this, but I think modern browsers create a document wrapper even for non-text types viewed in the browser, eg. if you open an image and then type `document` in the console.
It could therefore be possible to directly annotate image files and other embedded media

**Comment by BigBlueHat** [comment]
@ikreymer I'd like to wrap CSV's in http://maxogden.github.io/data-table/ too, but let's take it a thing at a time, eh? :wink:
